src/mpi/spawn/spawn_impl.c

```diff
@@ -78,7 +78,7 @@ static int MPIR_fd_recv(int fd, void *buffer, int length)
     return 0;
 }
 
-int MPIR_Comm_join(int fd, MPI_Comm * intercomm)
+int MPIR_Comm_join_impl(int fd, MPIR_Comm ** p_intercomm_ptr)
 {
```

Comment:
Can  you please move the refactoring of `MPIR_Comm_join` into a separate commit?

Reply:
Will do.

Reply:
done